recipes/py2neo/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name='py2neo' %} 
+{% set version='4.0.0b2' %}
+{% set sha256='a0635295a4399ae8023b6515c96ee6b2662bf682164277feb0a198e4f92cff7b' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # patch to fix writing to the console in unicode in python2/3 friendly way
+    - console.patch
```

Comment:
This is fixed in this (as yet unreleased) commit of the upstream project: https://github.com/technige/py2neo/commit/9b11afd303bd05544ceacb511001101d71e79e5b